### ISSUE TITLE
https://downloads.sentry-cdn.com/sentry-cli returns 403 preventing npm install

### ISSUE THREAD

**Issue by Cherry** [getsentry/sentry-cli#570]
https://downloads.sentry-cdn.com/sentry-cli is currently returning a 403 preventing our builds on CI, and preventing any local `npm install`.

**Comment by Cherry** [comment]
https://status.sentry.io/ reports "All Systems Operational", yet this is still failing to install. Please update ASAP.

**Comment by tonyo** [comment]
@Cherry What version are you trying to install?

> https://downloads.sentry-cdn.com/sentry-cli is currently returning a 403

It was always like that, the directory listing is disabled there.
Direct download links look like this: https://downloads.sentry-cdn.com/sentry-cli/1.47.0/sentry-cli-Darwin-x86_64

Also: older (pre 1.40?) binary versions of `sentry-cli` were pulled from GitHub releases, and GitHub had some issues today, might have been related:

<img width="1056" alt="image" src="https://user-images.githubusercontent.com/1120468/61649105-772ee500-acb9-11e9-8bae-e1718be43405.png">

**Comment by Cherry** [comment]
I'm seeing the error when installing `@sentry/webpack-plugin@1.7.0`, which is attempting to install `@sentry/cli@1.47.0`. 

Specifically:

```
> @sentry/cli@1.47.0 install F:\GitHub\website\node_modules\@sentry\cli
> node scripts/install.js

Error: Received 403: Forbidden

...

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @sentry/cli@1.47.0 install: `node scripts/install.js`
npm ERR! Exit status 1
```

I replicated the issue in a fresh directory by simply running:
`npm init -y && npm i @sentry/webpack-plugin@1.7.0`. Is `https://downloads.sentry-cdn.com/sentry-cli/1.47.0/sentry-cli-Windows-x86_64.exe` the correct Windows URL?

**Comment by tonyo** [comment]
> It appears that there wasn't a x64 exe released for the latest version.

You are right, our bad.
Should be fixed now.

**Comment by Cherry** [comment]
Thanks, I can confirm this is now resolved.

**Comment by preetjdp** [comment]
Getting a 403 while trying to download the sentry CLI binary, is there a work around available

https://downloads.sentry-cdn.com/sentry-cli/1.62.0/sentry-cli-Linux-arm64.

**Comment by kamilogorek** [comment]
@preetjdp there's no arm64 builds for linux yet, only for osx, see: https://sentry.io/get-cli/
Here's PR adding these builds (should be complete next week): https://github.com/getsentry/sentry-cli/pull/890

**Comment by andrewmclagan** [comment]
next week hey...

**Comment by kamilogorek** [comment]
ARM builds have been released as a part of 1.63.0, not sure what your comment refers to

**Comment by jozefvaclavik** [comment]
I'm getting same with `"sentry-cli": "^2.23.1"`

```sh
node scripts/install.js
Error: Unable to download sentry-cli binary from https://downloads.sentry-cdn.com/sentry-cli/1.63.0/sentry-cli-Linux-arm64.
Server returned 403: Forbidden.
```
